src/sentry/search/snuba/backend.py

```diff
@@ -122,6 +123,31 @@ def inbox_filter(inbox, projects):
     return query
 
 
+def owner_filter(owner, projects):
+    from sentry.models import Team, User
+
+    organization_id = projects[0].organization_id
+    project_ids = [p.id for p in projects]
+    if isinstance(owner, Team):
+        return Q(
+            id__in=GroupOwner.objects.filter(
+                team=owner, project_id__in=project_ids, organization_id=organization_id
+            )
+            .values_list("group_id", flat=True)
+            .distinct()
+        )
+    elif isinstance(owner, User):
+        return Q(
```

Comment:
if we want `owner:me`/`owner:<user>` to include team info like the assigned filter, do we also have to do a check for teams that are assigned to the user, or is that already handled here?

Reply:
Yeah, at first I wasn't sure if `owner:dwang@sentry.io` was going to include the team for that user - or if that was specific to `owner:me`. If it was specific, I think I would have needed to change the parser as it sends a User object whether you type `me` or an email and I'd have no way to distinguish. 

We decided to include the teams of a User for all owner searches and I added some logic to this function to handle that.